priv/perf/apps/load_test/lib/child_chain/utxos.ex

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.ChildChain.Utxos do
```

Comment:
I would put that into `LoadTest.Watcher....` module. It does interact with watcher API. If we want to keep it more general that child chain is the child chain app + watcher that's ok, and we don't need to use Watcher name in the module. I leave it up to you.

Reply:
Yes, I was thinking in the more general childchain case like you mentioned.
Not sure which is best though. I'll leave it like this for now, will think about it further.